doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_ckan_sink.md

```diff
@@ -35,31 +44,58 @@ This is done at the Cygnus Http listeners (in Flume jergon, sources) thanks to [
 ###<a name="section1.2"></a>Mapping Flume events to CKAN data structures
 [CKAN organizes](http://docs.ckan.org/en/latest/user-guide.html) the data in organizations containing packages or datasets; each one of these packages/datasets contains several resources whose data is finally stored in a PostgreSQL database (CKAN Datastore) or plain files (CKAN Filestore). Such organization is exploited by `NGSICKANSink` each time a Flume event is going to be persisted.
 
-According to the [naming conventions](naming_convetions.md), an organization called as the `fiware-service` header values is created (if not existing yet). The same occurs with a package/dataset named as the `fiware-servicePath` header value.
+[Top](#top)
+
+####<a name="section1.2.1"></a>Organizations naming conventions
+An organization named as the notified `fiware-service` header value (or, in absence of such a header, the defaulted value for the FIWARE service) is created (if not existing yet).
+
+[Top](#top)
+
+####<a name="section1.2.2"></a>Packages/datasets naming conventions
+A package/dataset named as the concatenation of the notified `fiware-service` and `fiware-servicePath` header values (or, in absence of such headers, the defaulted value for the FIWARE service and service path) is created (if not existing yet) in the above organization.
+
+Please observe if the notified/defaulted FIWARE service path is the root one, i.e. `/`, then the package/dataset name is equals to the organization name.
+
+The concatenation character is the underscore, `_`.
+
+[Top](#top)
+
+####<a name="section1.2.3"></a>Resources naming conventions
+CKAN resources follow a single data model (see the [Configuration](#section2.1) section for more details), i.e. per entity. Thus, a resource name always take the entity ID and type, concatenated by the underscore character, `_`. Such a name is already given in the `notified_entities`/`grouped_entities` header values (depending on using or not the grouping rules, see the [Configuration](#section2.1) section for more details) within the Flume event.
+
+It must be noticed a CKAN Datastore (and a viewer) is also created and associated to the resoruce above. This datastore, which in the end is a PostgreSQL table, will hold the persisted data.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1042/commits/3913c56cb58a1dea3f077c77817cdce9080c530f